src/coredump/coredumpctl.c

```diff
@@ -883,15 +907,30 @@ static int dump_core(int argc, char **argv, void *userdata) {
         return 0;
 }
 
-static int run_gdb(int argc, char **argv, void *userdata) {
+static int run_debug(int argc, char **argv, void *userdata) {
         _cleanup_(sd_journal_closep) sd_journal *j = NULL;
-        _cleanup_free_ char *exe = NULL, *path = NULL;
+        _cleanup_free_ char *exe = NULL, *path = NULL, *fork_name;
         bool unlink_path = false;
         const char *data;
         size_t len;
         pid_t pid;
         int r;
 
+        if (streq(argv[0], "gdb")) {
+                arg_debugger = "gdb";
+        } else if (arg_debugger == NULL) {
```

Comment:
no {} around single-line if blocks. See CODING_STYLE...

That said, as mentioned above I don't think `coredumpctl gdb` should behave any different from `debug` but be merely a hidden compat alias for it.